truss/truss_handle.py

```diff
@@ -388,6 +389,21 @@ def docker_build_setup(self, build_dir: Optional[Path] = None):
         image_builder.prepare_image_build_dir(build_dir)
         return image_builder.docker_build_command(build_dir)
 
+    def run_watch_server(
+        self,
+        build_dir: Optional[Path] = None,
+        work_dir: Optional[Path] = None,
+        port: Optional[int] = None,
+    ):
```

Comment:
Very minor: Will be good to add a comment or rename the function to reflect that this it reloads the local truss server, just for clarity. Because this watches and performs that specific action as a result.